tests/unit/test_protocols.py

```diff
@@ -99,6 +98,10 @@
     'rest-xml': RestXMLParser,
 }
 PROTOCOL_TEST_BLACKLIST = ['Idempotency token auto fill']
+IGNORE_LIST_FILENAME = "protocol-tests-ignore-list.json"
+PROTOCOL_TEST_IGNORE_LIST_PATH = os.path.join(TEST_DIR, IGNORE_LIST_FILENAME)
+with open(PROTOCOL_TEST_IGNORE_LIST_PATH) as f:
+    PROTOCOL_TEST_IGNORE_LIST = json.load(f)
```

Comment:
Do we use `PROTOCOL_TEST_IGNORE_LIST_PATH ` somewhere else? Should this whole thing be a self-contained function or `pytest` fixture instead?